src/main/java/seedu/address/logic/commands/MarkAttendanceCommand.java

```diff
@@ -57,13 +57,14 @@ public MarkAttendanceCommand(Name name, LocalDate date, Attendance attendance) {
     public CommandResult execute(Model model) throws CommandException {
         // Find the student by name
         Student student = model.getStudentByName(name);
-
         if (student == null) {
             throw new CommandException("Student not found: " + name);
         }
+        TutorialGroup tg = student.getTutorialGroup();
+
 
         // Mark attendance
         student.markAttendance(date, attendance.value);
```

Comment:
Is there a reason why this is done like this? It feels like it interchanges between the attendance object and status string a lot

Reply:
That's true. Will omit it